### ISSUE TITLE
Where is accessible name and descriptions defined?

### ISSUE THREAD

**Issue by zcorpan** [w3c/accname#34]
https://w3c.github.io/accname/#accessible-name-and-description-mapping says

> Information concerning name and description accessibility API mappings, including relationships, such as labelled-by/label-for and described-by/description-for, is documented in the Core Accessibility API Mappings specification [CORE-AAM-1.1].

which says

> The editor's draft of the Core Accessibility API Mappings 1.1 is [published from a new location](http://w3c.github.io/core-aam/)

ok. https://w3c.github.io/core-aam/#accessible-names-and-descriptions

> For more detail, see the [Accessible Name and Description Computation and API Mappings](https://w3c.github.io/aria/accname-aam/accname-aam.html) specification.

> The editor's draft of the Accessible Name and Description Computation 1.1 is [published from a new location](http://w3c.github.io/accname/) 

and we're back where we started. What's up?

**Comment by joanmarie** [comment]
I probably need more coffee, but I'm not sure what the question is. Are you saying there is something wrong with the redirects? Or is your question about the mappings? In the meantime....

The AccName spec defines the _platform-independent_ name and description _calculation_. To me, that is the answer to "Where is accessible name and descriptions defined?"

What is "documented in the Core Accessibility API Mappings specification" are the "accessibility API _mappings_." (emphasis added)

In other words, once you calculate the platform-independent name and description as per the AccName spec, you need to how you expose ("map") the results via platform accessibility APIs such at ATK, IAccessible2, UIAutomation, or nsAccessibility (aka AXAPI). Core AAM tells you how to do that platform-specific exposure.

Please let me know what information or changes you seek. Sorry for being dense and thanks for the report!

**Comment by zcorpan** [comment]
OK now I found this

https://w3c.github.io/accname/#mapping_additional_nd_name

> User agents MUST compute an accessible name using the rules outlined below in the section titled Text Alternative Computation.

so the algorithm is the one at

https://w3c.github.io/accname/#mapping_additional_nd_te

I filed this because I simply couldn't find what I was looking for and instead ended up going in circles.

**Comment by zcorpan** [comment]
So I think there are several things that could be improved.

* The links should be updated so they point to the new locations.
* The links could go to a specific section.
* The section title shouldn't be "Text Alternative Computation" if it's also used for accessible name computation; maybe "Text Alternative and Accessible Name Computation" (or something)
* The paragraphs that link to the other spec could clarify what the reader is expected to find in the spec they are currently reading vs what's in the other spec.

**Comment by joanmarie** [comment]
@zcorpan agreed re the language confusion. If the section title were "Name and Description Computation" rather than "Text Alternative Computation," would that be more clear to you?

**Comment by zcorpan** [comment]
Yes.

**Comment by zcorpan** [comment]
or "Accessible Name and Description Computation"

**Comment by zcorpan** [comment]
Thank you! I think those fixes are enough to close this issue.

**Comment by joanmarie** [comment]
> Thank you! I think those fixes are enough to close this issue.

Awesome. Thanks again for the report. Please re-open if you find something else related to this which we need to fix.
